libs/fairness/src/lib/WizardReport.tsx

```diff
@@ -342,51 +359,81 @@ export class WizardReport extends React.PureComponent<IReportProps, IState> {
+      const disparityBounds: Dictionary<IBounds> = {};
       const performance = await this.getMetric(
-        this.props.performancePickerProps.selectedPerformanceKey
+        this.props.performancePickerProps.selectedPerformanceKey,
+        this.props.errorPickerProps.selectedErrorKey
       );
       // TODO: extend disparities to query for all possible kinds of disparities
       // https://github.com/microsoft/responsible-ai-widgets/issues/65
+      const fairnessResponse: IFairnessResponse = await this.getFairnessMetric(
+        this.props.fairnessPickerProps.selectedFairnessKey,
+        fairnessOptions[this.props.fairnessPickerProps.selectedFairnessKey]
+          .fairnessMode,
+        this.props.errorPickerProps.selectedErrorKey
+      );
       disparities[this.props.fairnessPickerProps.selectedFairnessKey] =
-        await this.getFairnessMetric(
-          this.props.fairnessPickerProps.selectedFairnessKey,
-          fairnessOptions[this.props.fairnessPickerProps.selectedFairnessKey]
-            .fairnessMode
-        );
+        fairnessResponse.overall;
+      if (fairnessResponse.bounds) {
+        disparityBounds[this.props.fairnessPickerProps.selectedFairnessKey] =
+          fairnessResponse.bounds;
+      }
       switch (this.props.dashboardContext.modelMetadata.PredictionType) {
         case PredictionTypes.BinaryClassification: {
-          falseNegativeRates = await this.getMetric("miss_rate");
-          falsePositiveRates = await this.getMetric("fallout_rate");
-          outcomes = await this.getMetric("selection_rate");
+          falseNegativeRates = await this.getMetric(
+            "miss_rate",
+            this.props.errorPickerProps.selectedErrorKey
+          );
+          falsePositiveRates = await this.getMetric(
+            "fallout_rate",
+            this.props.errorPickerProps.selectedErrorKey
+          );
+          outcomes = await this.getMetric(
+            "selection_rate",
+            this.props.errorPickerProps.selectedErrorKey
+          );
           break;
         }
         case PredictionTypes.Probability: {
-          predictions =
-            this.props.dashboardContext.predictions[
-              this.props.selectedModelIndex
-            ];
-          overpredictions = await this.getMetric("overprediction");
-          underpredictions = await this.getMetric("underprediction");
-          outcomes = await this.getMetric("average");
+          predictions = this.props.dashboardContext.predictions[
+            this.props.selectedModelIndex
+          ];
+          overpredictions = await this.getMetric(
+            "overprediction",
+            this.props.errorPickerProps.selectedErrorKey
+          );
+          underpredictions = await this.getMetric(
+            "underprediction",
+            this.props.errorPickerProps.selectedErrorKey
+          );
+          outcomes = await this.getMetric(
+            // call will always return error bounds
```

Comment:
Not sure about this - a previously saved set of metrics may not have errors included.